src/ReachSets/ContinuousPost/BFFPS19/reach_helper.jl

```diff
@@ -0,0 +1,11 @@
+#check whether intersection with between cartesian product array and constraints list is empty
+@inline _is_disjoint(ca::CartesianProductArray, guards_constraints_proj::Vector{HPolyhedron{N}}) where {N} = any(c -> isdisjoint(ca, c), guards_constraints_proj)
```

Comment:
```suggestion
@inline _is_disjoint(X::LazySet{N}, sets::Vector{<:LazySet{N}}) where {N} =
    any(Y -> isdisjoint(X, Y), sets)
```
Actually, you can use `isdisjoint(X, UnionSetArray(sets))` provided by `LazySets`, which I find conceptually cleaner.